deploy/kubernetes/base/controller.yaml

```diff
@@ -32,7 +31,7 @@ spec:
           tolerationSeconds: 300
       containers:
         - name: ebs-plugin
-          image: k8s.gcr.io/provider-aws/aws-ebs-csi-driver:v0.10.0
+          image: k8s.gcr.io/provider-aws/aws-ebs-csi-driver:latest
```

Comment:
Hmm this is kind of dangerous. What if we start building alpha images?

Reply:
people should be using the stable overlay. However I agree best not to even mention latest tag, I'l lcheck where it's coming from (like is it  being propagated from a helm chart?)